File: C#/1015.cs
using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            string[] vet = Console.ReadLine().Split(' ');
            double pri = Double.Parse(vet[0]);
            double seg = Double.Parse(vet[1]);
            vet = Console.ReadLine().Split(' ');
            double ter = Double.Parse(vet[0]);
            double qua = Double.Parse(vet[1]);
            double resultado = Math.Sqrt((Math.Pow(ter - pri, 2)) + (Math.Pow(qua - seg, 2)));
            Console.WriteLine(string.Format("{0:0.0000}", resultado));
        }
    }
}

File: Python/1015.py
import math

p1 = input().split(" ")
p2 = input().split(" ")
result = math.sqrt(
    pow((float(p2[0]) - float(p1[0])), 2) +
    pow((float(p2[1]) - float(p1[1])), 2)
)
print('%.4f' % result)
